pkg/cmd/cli/install/install.go

```diff
@@ -0,0 +1,155 @@
+/*
+Copyright 2019 the Heptio Ark contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package install
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+
+	"k8s.io/client-go/dynamic"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	api "github.com/heptio/velero/pkg/apis/velero/v1"
+	"github.com/heptio/velero/pkg/client"
+	"github.com/heptio/velero/pkg/cmd"
+	"github.com/heptio/velero/pkg/cmd/util/output"
+	"github.com/heptio/velero/pkg/install"
+	"github.com/heptio/velero/pkg/util/logging"
+)
+
+// InstallOptions collects all the options for installing Velero into a Kubernetes cluster.
+type InstallOptions struct {
+	Namespace                 string
+	DeploymentName            string
```

Comment:
That's fine with me. I'll take another pass through the args and see if anything makes sense to eliminate, as I haven't really adjusted them since scoping the issue down.